learn/configuration/instance_options.md

```diff
@@ -371,14 +371,18 @@ Sets the maximum size of [accepted payloads](/learn/core_concepts/documents.md#d
 
 ### Schedule snapshot creation
 
-::: warning
-🚩 This option does not take any values. Assigning a value will throw an error. 🚩
-:::
-
 **Environment variable**: `MEILI_SCHEDULE_SNAPSHOT`
 **CLI option**: `--schedule-snapshot`
+**Default value**: `false` or `86400` (24 hours)
```

Comment:
I'm not sure if we can count the `86400` as a default value, given it is not what happens by default (by default, snapshots are disabled), but I agree it should happen *somewhere*, I'm just unsure under which form.